.bcr/presubmit.yml

```diff
@@ -0,0 +1,10 @@
+matrix:
+  platform:
+    - macos
+    - ubuntu2004
+tasks:
+  run_tests:
+    build_targets:
```

Comment:
missing: `platform: ${{ platform }}`

Also should we be testing against bazel 7 and 8 like the other repos?